src/app/detail/ui/styledComponent.tsx

```diff
@@ -91,12 +91,6 @@ export const TapWriteComment = styled(Tapwrite)(({ theme }) => ({
 }))
 
 export const TapWriteTaskEditor = styled(Tapwrite)(({ theme }) => ({
-  minHeight: '30vh',
+  minHeight: '10vh',
   fontSize: '20px',
 }))
```

Comment:
This is affecting the Tapwrite instance used in the details page as well. Please consider it.